### PR TITLE
Add ClickHouse plugin

### PR DESCRIPTION
**Pull request by edmundito**
## Change description

Adds ClickHouse plugin docs, integration page, and what's new.

image

image

image

### Development

- [x] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/docs/config/code-config/apps.mdx [hunk 1]
```diff
@@ -56,6 +56,7 @@ This will give you something like the following:
 
     bigquery:app (id) - Config for a Grouparoo bigquery App
     csv:app (id) - Config for a CSV App
+    clickhous:app (id) - Config for a ClickHouse App
```
Comment (teallarson) [inline pages/docs/config/code-config/apps.mdx]:
```suggestion
    clickhouse:app (id) - Config for a ClickHouse App
```

**Review by teallarson — COMMENTED** [review]
Looks good!  This matches how we have the other Plugin docs are written.  Maybe @evantahler or @bleonard can chime in -- we've had a story in the icebox for a while to transition Plugin docs to emphasize Config UI over CLI.  Do we want newly added plugin docs to move in that direction or continue how they have been for consistency?

File: README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ If you run into an issue installing Sharp on an Apple M1 machine, make sure `vip
 brew install vips
 ```
 
-If you find yourself needing to change the API endpoint that the webiste is using, you can use the `API_URL` environment variable, ie: `API_URL=http://localhost:8080`
+If you find yourself needing to change the API endpoint that the website is using, you can use the `API_URL` environment variable, ie: `API_URL=http://localhost:8080`
```
Comment (evantahler) [inline README.md:29]: 😓

File: data/plugins/clickhouse.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
+import { Plugin } from "./index";
+
+export const ClickHouse: Plugin = {
+  name: "ClickHouse",
+  description:
+    "Grouparoo's ClickHouse integration allows you to sync all of your customer data that lives in your ClickHouse database with various destinations such as CRMs, Customer Support Tools, and Marketing Tools.",
+  priority: 80,
```
Comment (evantahler) [inline data/plugins/clickhouse.ts:7]: This is interesting - I was wondering why Clickouse wasn't appearing as a source on the destination integration pages 
Screen Shot 2021-11-16 at 7 37 17 AM

But we only show the first 5 by priority.  https://github.com/grouparoo/www.grouparoo.com/blob/dd1f05b4576b4d79970e23d7c83d8df81e36fdbc/components/home/IntegrationSources.tsx#L17

This OK for now and we can re-shuffle the priorities later if we want

File: data/plugins/clickhouse.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      description: "HTTP Port on which to open database connection.",
+    },
+    database: {
+      required: true,
+      description: 'The database name - e.g. `"data_warehouse"`.',
+    },
+    user: {
+      default: undefined,
+      description:
+        'The user to connect to the database - e.g. `"grouparoo_user"`.  If you are connecting to localhost, leave as `undefined`.',
```
Comment (evantahler) [inline data/plugins/clickhouse.ts]: I think the default user is `default`

File: pages/whats-new/2021-11-15-clickhouse.mdx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+title: "ClickHouse Plugin"
+date: "2021-11-15"
+tags: ["new"]
+description: "Grouparoo can now use a BigQuery data warehouse to create profiles."
+github: "https://github.com/grouparoo/grouparoo/pull/2467"
+---
+
+With the release of Grouparoo 0.74 you can [sync data from and to
```
Comment (evantahler) [inline pages/whats-new/2021-11-15-clickhouse.mdx]: v0.7.4

**Review by evantahler — APPROVED** [review]
Nice!  You found all the places that a new plugin should be added, and user our data objects to load in the plugin.

Some minor updates to make below, and then this can be merged in!

**Comment by evantahler** [comment]
> Looks good! This matches how we have the other Plugin docs are written. Maybe @evantahler or @bleonard can chime in -- we've had a story in the icebox for a while to transition Plugin docs to emphasize Config UI over CLI. Do we want newly added plugin docs to move in that direction or continue how they have been for consistency?

Let's update all the plugins in the other story - for now, everything should match, as it does in this PR

File: data/plugins/clickhouse.ts (hunk 3 above)
Comment (edmundito) [reply data/plugins/clickhouse.ts:7]: Yep, I noticed that as well. I didn't want to prioritize clickhouse over the current ones being prioritized. I do think that should be a separate change

File: pages/whats-new/2021-11-15-clickhouse.mdx (hunk 5 above)
Comment (edmundito) [reply pages/whats-new/2021-11-15-clickhouse.mdx]: D'oh

**Pull request merged**